### ISSUE TITLE
Mouse clicks/highlights still showing even when turned off in settings

### ISSUE THREAD

**Issue by aalokt89** [wulkano/kap#231]
Platform: Mac, El Capitan
Step to reproduce:
-Open Kap preferences
-Turn off 'show cursor' and 'mouse clicks'
-Record screen

Current behaviour:
- Mouse cursor still appears
- Click highlights still appear

Expected behaviour:
- Mouse cursor and  clicks/highlights should not appear
![kapture 2017-09-13 at 12 20 34](https://user-images.githubusercontent.com/19478427/30388734-9afb8f02-987e-11e7-8b8d-1fd4244ee33e.gif)

**Comment by CodeTheory** [comment]
Thanks for the report. I managed to reproduce the issue.

### Steps to reproduce accurately (even on current master)
- Turn on both `Show mouse cursor` and `Highlight clicks`
- Turn off _just_ `Show mouse cursor` (which should turn off Highlight clicks in the UI)
- Record something and click around

![click-bug](https://user-images.githubusercontent.com/1695613/30389374-914a8ad6-98aa-11e7-86bd-9b9ffe79ae26.gif)

**Comment by alosarjos** [comment]
I've been checking this. I've been able to fix the highlight clicks, but the mouse cursor, I think it's something related to aperture. I just tried the example adding the options file on the Aperture project example

```javascript
const options = {
    fps: 30,
    showCursor: false
  };
```

and the cursor is still appearing. 

Still, I will do the pull request for click highlights, and once the mouse cursor is fixed on Aperture library, I think it should work.
